src/consumption.cpp

```diff
@@ -162,6 +248,15 @@ edible_rating player::can_eat( const item &food, bool interactive, bool force )
         return INEDIBLE;
     }
 
+    if( comest->comesttype == "FOOD" || comest->comesttype == "DRINK" ) {
+        for( const auto &m : food.type->materials ) {
+            if( !m.obj().edible() ) {
+                maybe_print( m_info, _( "That doesn't look edible in it's current form." ) );
```

Comment:
Should be "its", not "it's". The apostrophe is for "it is".

"The food item is not edible in its current form" vs "The food item is not edible, it's in a form that can't be digested".